src/meta-srv/src/metasrv.rs

```diff
@@ -40,6 +41,8 @@ pub struct MetaSrvOptions {
     pub datanode_lease_secs: i64,
     pub selector: SelectorType,
     pub use_memory_store: bool,
+    pub metrics_addr: String,
```

Comment:
question: Could we remove this field? We already have http_opts.addr

Reply:
yes, I will be remove this field.